integration-tests/java/src/integration-test/java/com/stellarstation/api/test/satellite/SatelliteStreamerTest.java

```diff
@@ -72,7 +72,8 @@ public void onNext(SatelliteStreamResponse response) {
         }
 
         if (response.hasReceiveTelemetryResponse()) {
-          ByteString data = response.getReceiveTelemetryResponse().getTelemetry().getData();
+          ByteString data =
```

Comment:
Should probably update this similarly to the printing client where all data is combined to a bytestring.